### PR TITLE
Unit_test_dassert_is_integer

### PR DESCRIPTION
**Pull request by jayati1397**
Created this PR for Issue [#1079](https://github.com/kaizen-ai/kaizenflow/issues/1079).

**Changes Made:**
- Implemented unit tests for dassert_is_integer.

**Testing:**
Ran code through pylint and pytest.
- Pylint Errors: Class name: currently follows the convention specified [here](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/coding/all.write_unit_tests.how_to_guide.md#naming-and-placement-conventions).

```
OUTPUT from the added lines in the file
helpers/test/test_dbg.py:719:0: C0115: Missing class docstring (missing-class-docstring)
helpers/test/test_dbg.py:719:0: C0103: Class name "Test_dassert_is_integer" doesn't conform to PascalCase naming style (invalid-name)
```
- Pytest: No errors.
```
OUTPUT:
collected 5 items                                                                                                                                       

helpers/test/test_dbg.py::Test_dassert_is_integer::test1 (0.00 s) PASSED [ 25%]
helpers/test/test_dbg.py::Test_dassert_is_integer::test2 (0.00 s) PASSED [ 50%]
helpers/test/test_dbg.py::Test_dassert_is_integer::test3 (0.00 s) PASSED [ 75%]
helpers/test/test_dbg.py::Test_dassert_is_integer::test4 (0.00 s) PASSED [100%]
```

@samarth9008 please let me know if you have any further suggestions

### CODE REVIEW

**Review by samarth9008 — COMMENTED** [review]
1 nit and LG. Good PR.

**Review by gpsaggese — COMMENTED** [review]
LGTM

File: helpers/test/test_dbg.py [hunk 1]
```diff
@@ -713,3 +713,51 @@ def test4(self) -> None:
...
+    def test1(self) -> None:
+        """
+        Test that the function do not raise the exception with integer values.
+        """
+        val = 5
+        hdbg.dassert_is_integer(val)
+
+    def test2(self) -> None:
+        """
+        Test that the function do not raise the exception with float values that represent an integer.
```
Comment (gpsaggese) [inline helpers/test/test_dbg.py:730]: linter

Comment (jayati1397) [reply helpers/test/test_dbg.py]: ran the linter and pushed the changes after linter was ran

**Review by samarth9008 — APPROVED** [review]
LG, merging.

**Pull request merged**